core/src/main/java/org/opensearch/sql/expression/datetime/DateTimeFunction.java

```diff
@@ -1136,6 +1154,22 @@ private ExprValue exprFromUnixTimeFormat(ExprValue time, ExprValue format) {
     return DateTimeFormatterUtil.getFormattedDate(value, format);
   }
 
+  /**
+   * get_format implementation for ExprValue.
+   *
+   * @param type ExprValue of the type.
+   * @param format ExprValue of Time/String type
+   * @return ExprValue..
+   */
+  private ExprValue exprGetFormat(ExprValue type, ExprValue format) {
+    String key = type.stringValue() + "-" + format.stringValue();
```

Comment:
Does https://guava.dev/releases/19.0/api/docs/com/google/common/collect/Table.html solve the problem without concat the string value?

Reply:
Fixed [here](https://github.com/opensearch-project/sql/pull/1299/files#:~:text=//%20Map%20used%20to,.build()%3B), and [here](https://github.com/opensearch-project/sql/pull/1299/files#:~:text=private%20ExprValue%20exprGetFormat,.of()%3B)